plugins/log/go.mod

```diff
@@ -2,7 +2,4 @@ module github.com/extism/extism-sdk-plugins-log
 
 go 1.20
 
-require (
-	github.com/extism/go-pdk v1.0.0-rc1.0.20231019212020-62d54a6e0263 // indirect
-	github.com/valyala/fastjson v1.6.3 // indirect
-)
+require github.com/extism/go-pdk v1.0.7-0.20240804170849-7cb3d48ce7cc
```

Comment:
pointing to this: https://github.com/extism/go-pdk/commit/7cb3d48ce7ccff27c3eab9edb251d56caabb5c18